examples/rbfe_edge_list.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import csv
+import pickle
+
+import numpy as np
+from rdkit import Chem
+
+Chem.SetDefaultPickleProperties(Chem.PropertyPickleOptions.AllProps)
+
+from timemachine.constants import KCAL_TO_KJ
+from timemachine.fe import atom_mapping
+from timemachine.fe.rbfe import HostConfig, estimate_relative_free_energy
+from timemachine.fe.utils import get_mol_name
+from timemachine.ff import Forcefield
+from timemachine.md import builders
+from timemachine.parallel.client import CUDAPoolClient
+
+
+def get_mol_by_name(mols, name):
+    for m in mols:
+        if m.GetProp("_Name") == name:
+            return m
+
+    assert 0, "Mol not found"
+
+
+def run_pair(mol_a, mol_b, core, forcefield_path, protein_path, n_frames, seed):
+
+    box_width = 4.0
+    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(box_width)
+    solvent_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes, deboggle later
+    solvent_host_config = HostConfig(solvent_sys, solvent_conf, solvent_box)
+
+    forcefield = Forcefield.load_from_file(forcefield_path)
+
+    solvent_res = estimate_relative_free_energy(
+        mol_a, mol_b, core, forcefield, solvent_host_config, seed, n_frames=n_frames, prefix="solvent"
+    )
+
+    complex_sys, complex_conf, _, _, complex_box, complex_top = builders.build_protein_system(protein_path)
+    complex_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes, deboggle later
+    complex_host_config = HostConfig(complex_sys, complex_conf, complex_box)
+
+    complex_res = estimate_relative_free_energy(
+        mol_a, mol_b, core, forcefield, complex_host_config, seed + 1, n_frames=n_frames, prefix="complex"
+    )
+
+    return solvent_res, solvent_top, complex_res, complex_top
+
+
+def read_from_args():
+
+    parser = argparse.ArgumentParser(
+        description="Estimate relative free energy difference between complex and solvent given two ligands mol_a and mol_b."
```

Comment:
Update description (looks like this was copied from the `relative_free_energy.py` example?)

Reply:
willfix